### PR TITLE
Swedish localization

### PR DESCRIPTION
**Pull request by joforsell**

### CODE REVIEW

File: Localizations/SVRD/done/sv.xcloc/Localized Contents/en.xliff [hunk 1]
```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd">
+  <file original="SVRD/en.lproj/InfoPlist.strings" source-language="en" target-language="en" datatype="plaintext">
```
Comment (emin-grbo) [inline Localizations/SVRD/done/sv.xcloc/Localized Contents/en.xliff]:
```suggestion
  <file original="SVRD/en.lproj/InfoPlist.strings" source-language="en" target-language="sv" datatype="plaintext">
```

File: Localizations/SVRD/done/sv.xcloc/Localized Contents/en.xliff [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    </header>
+    <body>
+      <trans-unit id="CFBundleName" xml:space="preserve">
+        <source>SVRD</source>
+        <target>SVRD</target>
+        <note>Bundle name</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="SVRD/en.lproj/Localizable.strings" source-language="en" target-language="en" datatype="plaintext">
```
Comment (emin-grbo) [inline Localizations/SVRD/done/sv.xcloc/Localized Contents/en.xliff]:
```suggestion
  <file original="SVRD/en.lproj/Localizable.strings" source-language="en" target-language="sv" datatype="plaintext">
```

File: Localizations/SVRD/done/sv.xcloc/contents.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "developmentRegion" : "en",
+  "project" : "SVRD.xcodeproj",
+  "targetLocale" : "en",
```
Comment (emin-grbo) [inline Localizations/SVRD/done/sv.xcloc/contents.json]:
```suggestion
  "targetLocale" : "sv",
```

**Review by emin-grbo — CHANGES_REQUESTED** [review]
Hey there! Thank you so much for posting the PR.
Unfortunately because I provided the english as a template, you need to make some changes.

Just accepting these changes should work, after which i can merge all of it 🙌

**Comment by joforsell** [comment]
No problem! I'll fix it as soon as I figure this PR thing out 😅 Good learning experience for me! 😁

**Comment by emin-grbo** [comment]
Oh, as I added the corrections already, all you need to do is actually tap "commit suggestion" button :) 
I could actually do it myself of course, but if you want to see how it works, I will leave it like so for now 👍

**Comment by joforsell** [comment]
I see! Couldn't find them on the GitHub iOS app. I guess I learned something new today! :)

**Comment by emin-grbo** [comment]
superb! We should be fine now. Will try it out!

**Review by emin-grbo — APPROVED** [review]

**Pull request merged**